evap/evaluation/models.py

```diff
@@ -916,14 +971,31 @@ def send_to_user(self, user, courses=None, cc=True):
         else:
             cc_addresses = []
 
+        grade_documents = len(grade_document_courses) > 0
+        evaluation_results = len(evaluation_results_courses) > 0
```

Comment:
these names sound like lists but they aren't, please think of something else. maybe prepend with "has_"